src/main/solr/dssolr/conf/solrconfig.xml

```diff
@@ -811,6 +811,50 @@
          
     </requestHandler>
 
+
+
+
+  <!-- The SuggestComponent in Solr provides users with automatic suggestions for query terms.
+       You can use this to implement a powerful auto-suggest feature in your search application.
+       As with the rest of this solrconfig.xml file, the configuration of this component is purely
+       an example that applies specifically to this configset and example documents.
+
+       See: https://solr.apache.org/guide/solr/latest/query-guide/suggester.html
+    -->
+
+
+
+<!-- 
+   suggest for title field. If more fields are needed for suggest compontent the best way is to use copy-field and use that field.
+  Important: It is case sensitive now
+  Important: Suggest index will be build on start-up now. Change for large index
+  
+  Example calls:
+  http://devel11:10007/solr/ds/suggest?suggest.dictionary=dr_title_suggest&suggest.q=tv
+  http://devel11:10007/solr/ds/suggest?suggest.dictionary=dr_title_suggest&suggest.q=Kendtes  
```

Comment:
Sorry, no internal names allowed. A big shame as I love having demo links in the comments.